docs/dev.md

```diff
@@ -82,11 +82,38 @@ To properly deploy dev cluster you need to have the following variable set:
 
 ### Adopted Cluster Setup
 
-To "adopt" an existing cluster first obtain the kubeconfig file for the cluster. 
+To "adopt" an existing cluster first obtain the kubeconfig file for the cluster.
 Then set the `DEV_PROVIDER` to "adopted". Export the kubeconfig file as a variable by running the following:
 
 `export KUBECONFIG_DATA=$(cat kubeconfig | base64 -w 0)`
 
+### OpenStack Provider Setup
```

Comment:
Move this above the Adopted Cluster Setup section.

Reply:
sounds good